pkg/bench/rttanalysis: TestBenchmarkExpectation/DropRole/drop_3_roles failed

`pkg/bench/rttanalysis.TestBenchmarkExpectation/DropRole/drop_3_roles` [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/11199965?buildTab=tests&suite=github.com%2Fcockroachdb%2Fcockroach%2Fpkg%2Fbench%2Frttanalysis%3A+&expandedTest=build%3A%28id%3A11199943%29%2Cid%3A141486) on release-23.1 @ 5230230a98d2669b294bc564bc2d5c1b1fbd2f88

```
TestBenchmarkExpectation/DropRole/drop_3_roles
[02:25:07 ](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/11199965?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true&showLog=11199943_168231_168231&logFilter=debug&logView=flowAware)      Failed
[02:25:07 ](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_BazelEssentialCi/11199965?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true&showLog=11199943_168232_168232&logFilter=debug&logView=flowAware)      === RUN   TestBenchmarkExpectation/DropRole/drop_3_roles
      === CONT  TestBenchmarkExpectation/DropRole/drop_3_roles
          rtt_analysis_bench.go:206: DropRole/drop_3_roles: got 40.5, expected 42
          rtt_analysis_bench.go:212: wrote jaeger trace to /artifacts/tmp/_tmp/1b686e68658fd1761e16df16f8888527/logTestBenchmarkExpectation3986503865/DropRole_drop_3_roles.jaeger.json
              --- FAIL: TestBenchmarkExpectation/DropRole/drop_3_roles (141.60s)
```

Jira issue: CRDB-30414

--- a/pkg/bench/rttanalysis/testdata/benchmark_expectations
+++ b/pkg/bench/rttanalysis/testdata/benchmark_expectations
@@ -41,7 +41,7 @@ exp,benchmark
 17,DropDatabase/drop_database_3_tables
 26,DropRole/drop_1_role
 36,DropRole/drop_2_roles
-45,DropRole/drop_3_roles
+42-45,DropRole/drop_3_roles
 15,DropSequence/drop_1_sequence
 17,DropSequence/drop_2_sequences
 19,DropSequence/drop_3_sequences

